mod/groups/start.php

```diff
@@ -362,7 +362,7 @@ function groups_entity_menu_setup($hook, $type, $return, $params) {
 
 	/* @var ElggMenuItem $item */
 	foreach ($return as $index => $item) {
-		if (in_array($item->getName(), array('access', 'likes', 'unlike', 'edit', 'delete'))) {
+		if (in_array($item->getName(), array('access', 'edit', 'delete'))) {
```

Comment:
this should have been left intact... the menu items were remove probably for aesthetic reasons and not to 'disable' liking on groups (as that is current possible with stock Elgg, but only via the river entity menu).